cmd/zed/zed.d/zed-functions.sh

```diff
@@ -29,8 +30,8 @@ ZED_FLOCK_FD=8
 #
 zed_check_cmd()
 {
-    local cmd
-    local rv=0
+    cmd=$*
+    rv=0
```

Comment:
`dash`, `bash`, and `ash` all support the local keyword.  So even though this isn't strictly POSIX sh it's OK for us to leave this as is.  Let' not make any of these local keyword changes.  I should add that leaving these as local doesn't generate any shellcheck errors for me with shellcheck-0.3.7.  If you're getting warnings let's exclude them with `shellcheck -e`.